src/cortxfs/CMakeLists.txt

```diff
@@ -10,6 +10,14 @@ include_directories(${CORTXUTILSINC})
 set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${CORTX_CFLAGS}")
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${CORTX_CFLAGS}")
 
+# Turns on ADDB-based TDSB wrappers.
+# When this flag is disabled, KVSFS will be used the no-oop
```

Comment:
nitpick: Correct `will be used the` however you think but this seems in correct to me. Also two `be` on line 16-17. If it same for other MRs too then correct there also.